mne/io/nicolet/nicolet.py

```diff
@@ -13,11 +13,11 @@
 from ..constants import FIFF
 
 
-def read_raw_nicolet(input_fname, montage=None, eog=None, misc=None, ecg=None,
-                     emg=None, preload=False, verbose=None):
+def read_raw_nicolet(input_fname, ch_type, montage=None, eog=[], ecg=[],
+                     emg=[], misc=[], preload=False, verbose=None):
```

Comment:
maybe a nitpick but it would be nice to avoid passing mutable params.

maybe [] -> (,) to have a tuple?